src/components/screens/InfoCard.js

```diff
@@ -73,6 +74,18 @@ export const InfoCard = ({ addresses, category, contact, contacts, webUrls }) =>
         <RegularText>{category.name}</RegularText>
       </InfoBox>
     )}
+    {!!lengthKm && (
+      <InfoBox>
+        <RNEIcon
+          name="map"
+          type="material"
+          color={colors.primary}
+          iconStyle={{ marginRight: normalize(10) }}
```

Comment:
now as we have this iconStyle multiple times in this file, it would be nicer to have a const styles = ... for that, which we can use everywhere here for icons.
it helps for the future if we want to change the margin for example from 10 to something else. now we would need to change it on multiple places. if we would have a const for that particular style, we only need to change it on one place.